sdks/python/apache_beam/io/external/xlang_kafkaio_it_test.py

```diff
@@ -94,6 +94,7 @@ def run_xlang_kafkaio(self, pipeline):
     pipeline.run(False)
 
 
+@unittest.skip('BEAM-10663')
```

Comment:
```suggestion
@unittest.skip('BEAM-10663, BEAM-6868')
```

Reply:
If we add BEAM-6868 it should be in a skipIf that checks for the flink runner